tools/build-manylinux.sh

```diff
@@ -89,12 +89,13 @@ for PYBIN in /opt/python/cp*/bin/; do
     for PYTHON_VERSION in "${PYTHON_VERSIONS[@]}"; do
         if [[ $PYBIN == *"$PYTHON_VERSION"* ]]; then
             echo "## Installing $PYBIN"
-            "${PYBIN}/pip" install confluent_kafka -f /io/wheelhouse
-            "${PYBIN}/python" -c 'import confluent_kafka; print(confluent_kafka.libversion())'
-            "${PYBIN}/pip" install -r /io/tests/requirements.txt
-            "${PYBIN}/pytest" /io/tests/test_Producer.py
+            "${PYBIN}pip" -V
```

Comment:
Would be better avoiding the double slash by removing final slash from `for PYBIN in /opt/python/cp*/bin/; do`

As in `# Compile wheels`